src/backend/distributed/commands/function.c

```diff
@@ -1443,10 +1443,18 @@ DefineAggregateStmtObjectAddress(Node *node, bool missing_ok)
 	ObjectWithArgs *objectWithArgs = makeNode(ObjectWithArgs);
 	objectWithArgs->objname = stmt->defnames;
 
-	FunctionParameter *funcParam = NULL;
-	foreach_ptr(funcParam, linitial(stmt->args))
+	if (stmt->args != NIL)
+	{
+		FunctionParameter *funcParam = NULL;
+		foreach_ptr(funcParam, linitial(stmt->args))
+		{
+			objectWithArgs->objargs = lappend(objectWithArgs->objargs,
+											  funcParam->argType);
+		}
+	}
+	else
 	{
-		objectWithArgs->objargs = lappend(objectWithArgs->objargs, funcParam->argType);
+		objectWithArgs->objargs = list_make1(makeTypeName("anyelement"));
 	}
 
 	return FunctionToObjectAddress(OBJECT_AGGREGATE, objectWithArgs, missing_ok);
```

Comment:
Can you please update the comment of FunctionToObjectAddress to include aggregates as well? Please also check other functions used by FunctionToObjectAddress to make sure that we are mentioning about aggregates.